studybuddies/src/app/services/httperror-interceptor.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable, OnInit } from '@angular/core';
+import {HttpHandler, HttpRequest, HttpInterceptor, HttpErrorResponse} from '@angular/common/http';
+import { throwError } from 'rxjs';
+import { catchError } from 'rxjs/internal/operators';
+import { ErrorPageComponent } from '../components/error-page/error-page.component'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202842)
```suggestion
import { ErrorPageComponent } from "../components/error-page/error-page.component"
```